infra/docker/app/env.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -e
 
-./import-meta-env -x .env.example -p public/index.html
+./import-meta-env -x .env.local.defaults -p public/index.html
```

Comment:
C'était limite plus clair de garder `.env.example` je trouve, c'est plus conforme à l'usage de cette lib ("The .env example file path to load") et on ne manipule pas un `.local` dans le Dockerfile + env.sh